content/release-notes/1.20.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-10-06
+linktitle: "1.20.0"
+title: "1.20.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Disconnected nodes can be deleted from the cluster management page of the console on embedded clusters.
+* [Collect](https://troubleshoot.sh/docs/collect/), [Analyze](https://troubleshoot.sh/docs/analyze/), and [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) specs now support template functions.
+* Sequence, Cursor, ChannelName, VersionLabel, ReleaseNotes, and IsAirgap [template functions](/reference/template-functions/license-context/).
+* Applications installed into airgapped instances can now be upgraded using [KOTS CLI](/kots-cli/upstream/) -- NEEDS DOC
+* Support for headless embedded cluster installs. -- NEED DOC
+* Private images used by SupportBundle and Preflight [collectors](https://troubleshoot.sh/docs/collect/collectors/) are now automatically rewritten so they can be pulled in side the cluster.
+{{</features>}}
+
+{{<changes>}}
+* New [Version History](/kotsadm/updating/updating-kots-apps/#checking-for-updates) page design.
```

Comment:
Likewise, updated design for app dashboard version card.

Reply:
If there is an update we're showing latest available version.